gradle.properties

```diff
@@ -11,7 +11,7 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-VERSION_NAME=2.0.0
+VERSION_NAME=2.0.1
```

Comment:
no need to change that

Reply:
4d9dc5df4f6312c629c75e5aa2fd0148ef407b9e